UnitsNet.Tests/CustomCode/DensityTests.cs

```diff
@@ -132,12 +138,5 @@ public void ExpectDensityConvertedToMolarityCorrectly()
             var molarity = density.ToMolarity(mw).MolesPerCubicMeter;
             AssertEx.EqualTolerance(1026.98355, molarity, KilogramsPerCubicMeterTolerance);
         }
-
-        [Fact]
-        public void DensityTimesSpeedEqualsMassFlux()
```

Comment:
Did you intentionally remove this code?

Reply:
No, it was a mistake.
I restored the unit test.